website/docs/concepts/examples/cryptocurrency.md

```diff
@@ -450,7 +454,9 @@ TRANSACTION ID HERE ] }` in order to see the error.
 ```all
```

Comment:
do we want to remove all instances of ```all ?

Reply:
Okay, I added support for xml and changed it in the smart-functions file AND the installation file are well